components/date-picker/demo/presetted-ranges.ts

```diff
@@ -7,7 +7,7 @@ import endOfMonth from 'date-fns/endOfMonth';
     <nz-range-picker [nzRanges]="ranges1" ngModel (ngModelChange)="onChange($event)"></nz-range-picker>
```

Comment:
ranges1 => ranges

Reply:
Sorry I didn't see that one. But now it is fixed.